.github/workflows/at_server.yaml

```diff
@@ -515,7 +530,7 @@ jobs:
   push_canary_secondary_image:
     # Runs only after functional tests are completed.
     needs: [ unit_tests, functional_tests, end2end_test_12, end2end_test_34, end2end_test_56 ]
-    if: ${{ github.repository == 'atsign-foundation/at_server' && github.event_name == 'push' && contains(github.ref, 'refs/tags/c') }}
+    if: ${{ github.repository == 'atsign-foundation/at_server' && github.event_name == 'push' && contains(github.ref, 'refs/tags/c') || github.event.inputs.rollback_canary_secondary_image == 'true' }}
```

Comment:
In the steps below it shouldn't be building a new Docker image. What needs to happen instead is that the appropriate tag gets pointed back to the selected previous version.

So let's say that my latest release is `v3.0.18` and I want to roll back to `v3.0.17` the script needs to create and push manifest for the `prod` tag that's the same as the `prod-v3.0.17` manifest.